logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -180,6 +184,18 @@ internal class ConversationDataSource internal constructor(
         return fetchAllConversationsFromAPI()
     }
 
+    // TODO temporary method until backend API changed
+    override suspend fun fetchGlobalTeamConversation(): Either<CoreFailure, Unit> =
```

Comment:
The backend will change so that's not necessary to explicitly fetch the global team conversation.